pygeos/constructive.py

```diff
@@ -50,6 +50,9 @@ def boundary(geometry, **kwargs):
     ----------
     geometry : Geometry or array_like
         This function will return None for geometrycollections.
+    **kwargs
+        For other keyword-only arguments, see the
+        `Numpy ufunc docs <https://numpy.org/doc/stable/reference/ufuncs.html#ufuncs-kwargs>`_.
```

Comment:
```suggestion
        `NumPy ufunc docs <https://numpy.org/doc/stable/reference/ufuncs.html#ufuncs-kwargs>`_.
```

(very small nitpick, but with a search/replace using the full line it might be easy to change)

I think in general it's either all lowercase (the package), or if referrring the project they capitalize the P as well